Pod/Classes/NSAttributedString+BonMotUtilities.h

```diff
@@ -0,0 +1,15 @@
+//
+//  NSAttributedString+BonMotUtilities.h
+//  BonMot
+//
+//  Created by Eliot Williams on 3/9/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSAttributedString (BonMotUtilities)
+
+@property (nonatomic, readonly, nonnull) NSString *humanReadableString;
```

Comment:
#req category methods must be prefixed. Prefixing is annoying, and we could discuss other ways to accomplish this (a C function, for example, might make sense here), but if it's a category method (implicit with a property), it should be prefixed. In this case, I think the `bon_` prefix would actually make sense, because `humanReadableString` could mean different things in different contexts. I wonder whether a more specific name like "debuggingString` or similar might be more applicable?